install_config/upgrading/automated_upgrades.adoc

```diff
@@ -202,61 +221,155 @@ on what features are enabled.
 `openshift-enterprise`.
+
 . If you have multiple masters configured and want to enable rolling, full system
-restarts of the hosts, you can set the `*openshift_rolling_restart_mode*`
+restarts of the hosts, you can set the `openshift_rolling_restart_mode`
 parameter in your inventory file to `system`. Otherwise, the default value
 *services* performs rolling service restarts on HA masters, but does not reboot
 the systems. See
 xref:../install/advanced_install.adoc#configuring-cluster-variables[Configuring
 Cluster Variables] for details.
-. Run the *_v3_3_* upgrade playbook. If your inventory file is located somewhere
-other than the default *_/etc/ansible/hosts_*, add the `-i` flag to specify the
-location. If you previously used the `atomic-openshift-installer` command to run
-your installation, you can check *_~/.config/openshift/hosts_* (previously
-located at *_~/.config/openshift/.ansible/hosts_*) for the last inventory file
-that was used, if needed.
+
+. If you are upgrading from {product-title} 3.3 to 3.4, etcd must be upgraded from
+v2 to v3. {product-title} 3.4, however, continues to use the etcd v2 API, which
+is backwards compatible with etcd v3.
++
+To upgrade etcd, run the *_upgrade_etcd.yml_* Ansible playbook. The playbook
+checks for hosts defined in *[etcd]* group and upgrades etcd on those hosts.
++
+If your inventory file is located somewhere other than the default
+*_/etc/ansible/hosts_*, add the `-i` flag to specify the location. If you
+previously used the `atomic-openshift-installer` command to run your
+installation, you can check *_~/.config/openshift/hosts_* for the last inventory
+file that was used, if needed.
 +
 ----
-# ansible-playbook [-i </path/to/inventory/file>] \
-    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/upgrades/v3_3/upgrade.yml
+$ ansible-playbook -i <path/to/inventory/file> \
+    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/upgrades/upgrade_etcd.yml
 ----
-include::install_config/upgrading/automated_upgrades.adoc[tag=automated_upgrade_after_reboot]
 
-[[upgrading-to-ocp-3-3-asynchronous-releases]]
-=== Upgrading to {product-title} 3.3 Asynchronous Releases
-
-To apply
-xref:../../release_notes/ocp_3_3_release_notes.html#ocp-33-asynchronous-errata-updates[asynchronous errata updates] to an existing {product-title} 3.3 cluster:
+. At this point, you can choose to run the upgrade in a single or multiple phases.
+See xref:upgrading-control-plane-nodes-separate-phases[Upgrading the Control
+Plane and Nodes in Separate Phases] for more details which components are
+upgraded in each phase.
++
+[NOTE]
+====
+For either option, see xref:customizing-node-upgrades[Customizing Node Upgrades]
+for any `<customized_node_upgrade_variables>` shown below, if desired.
+====
++
+*Option A) Upgrade masters and nodes in a single phase.*
++
+Run the *_upgrade.yml_* playbook to upgrade the cluster in one phase: master
+components first, then nodes in-place:
```

Comment:
May want to consider referring to this as the control plane, rather than master components. Not sure if now is a good time or sometime in future. Control plane matches the terminology used upstream.